documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,85 @@
+.. _generating-necessary-inputs:
+
+Generate the necessary input files
+#############################################################
+
+The ``sct_compute_compression`` command uses the shape metric of the spinal cord a the level of compression and computes a ratio with healthy levels above and below the compression site.
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_compression_labels.nii.gz``)
```

Comment:
Again, we should use the `label-compression.nii.gz` suffix to be in line with the [intranet](https://intranet.neuro.polymtl.ca/data/dataset-curation.html#derivatives-structure).